DOCUMENTATION.md

```diff
@@ -104,751 +29,740 @@ import uk.gov.service.notify.SendSmsResponse;
+
+```
+String templateId="f33517ff-2a88-4f6e-b855-c550268ce08a";
 ```
 
-Otherwise the client will raise a `NotificationClientException`:
+#### personalisation (optional)
 
-|status code|error message|
-|:---|:---|
-|`429`|`429 {`<br>`"errors":`<br>`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type live of 60 requests per second"`<br>`}]`<br>`}`|
-|`429`|`429 {`<br>`"errors":`<br>`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (50) for today"`<br>`}]`<br>`}`|
-|`400`|`400 {`<br>`"errors":`<br>`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can"t send to this recipient using a team-only API key"`<br>`}]`<br>`}`|
-|`400`|`400 {`<br>`"errors":`<br>`[{`<br>`"error": "BadRequestError",`<br>`"message": ""Can"t send to this recipient when service is in trial mode - see https://www.notifications.service.gov.uk/trial-mode""`<br>`}]`<br>`}`|
+If a template has placeholder fields for personalised information such as name or reference number, you must provide their values in a map. For example:
 
-</details>
+```java
+Map<String, String> personalisation = new HashMap<>();
+personalisation.put("first_name", "Amala");
+personalisation.put("application_date", "2018-01-01");
+```
+If a template does not have any placeholder fields for personalised information, you must pass in an empty map or `null`.
 
-#### Arguments
+#### reference (optional)
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+A unique identifier. This reference can identify a single unique notification or a batch of multiple notifications. If you do not have a reference, you must pass in an empty string or `null`.
 
-##### `mobileNumber`
-The mobile number the SMS notification is sent to.
+```
+String reference='STRING';
+```
 
-##### `templateId`
+#### smsSenderId (optional)
 
-The template id is visible on the template page in the application.
+A unique identifier of the sender of the text message notification. To set this up:
 
-##### `reference`
-An optional unique identifier for the notification or an identifier for a batch of notifications. `reference` can be an empty string or null.
+1. Sign into your GOV.UK Notify account.
+1. Go to __Settings__.
+1. If you are not in the correct service, select __Switch service__ in the top right corner of the screen and select the correct one.
+1. Go to the __Text Messages__ section and select __Manage__ on the "Text Message sender" row.
+1. You can do one of the following:
+  - copy the ID of the sender you want to use and paste it into the method
+  - select __Change__ to change the default sender that the service will use, and select __Save__
 
-##### `personalisation`
+```
+String smsSenderId='8e222534-7f05-4972-86e3-17c5d9f894e2'
+```
+
+If you do not have have an `smsSenderId`, you must pass in an empty string or `null`.
+
+### Response
+
+If the request to the client is successful, you will receive the following `SendSmsResponse`:
```

Comment:
Maybe this will read better as:
... you will get a `SendSmsResponse` with the following attributes: